chips/sam4l/src/adc.rs

```diff
@@ -408,12 +447,19 @@ impl hil::adc::Adc for Adc {
             self.active.set(true);
             self.continuous.set(true);
 
+            let trgsel;
+            if self.cpu_clock.get() {
+                trgsel = 1; // internal timer trigger
+            } else {
+                trgsel = 3; // continuous mode
```

Comment:
I think this is the only unresolved comment. @brghena/@petarpenkov  thoughts still? Does this still need to be addressed?